include/tk2CMSSW_datatypes.hh

```diff
@@ -263,6 +263,7 @@ namespace insur {
         std::string childname;
         bool isDiskAtPlusZEnd;
         int numModules;
+        int discNumber;
```

Comment:
Why do you need `discNumber` here? We usually use `layerNumber` to count discs in the forward